### PR TITLE
ammonite-repl 0.5.8

### PR DESCRIPTION
**Pull request by emanresusername**
updates ammonite-repl version to 0.5.8

### CODE REVIEW

File: Formula/ammonite-repl.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
 class AmmoniteRepl < Formula
   desc "Ammonite is a cleanroom re-implementation of the Scala REPL"
   homepage "https://lihaoyi.github.io/Ammonite/#Ammonite-REPL"
-  url "https://git.io/vVfy5", :using => :nounzip
-  version "0.5.7"
-  sha256 "ef410dc843391c6b3199f2552a24f5fe3f3c79a7558d813089c8dce8b4ab97b6"
+  url "https://git.io/vro0u", :using => :nounzip
```
Comment (MikeMcQuaid) [inline Formula/ammonite-repl.rb]: Please expand this to the full URL, thanks!

Comment (MikeMcQuaid) [inline Formula/ammonite-repl.rb]: Also, what's the GitHub repo for this?

**Comment by emanresusername** [comment]
the github repo is https://github.com/lihaoyi/Ammonite/

**Comment by UniqMartin** [comment]
Merged. Thank you for this contribution to Homebrew, @emanresusername! :tada:

**Comment by emanresusername** [comment]
gif-keyboard-16517529264368997446

**Pull request merged**